pysph/examples/gas_dynamics/riemann_2d.py

```diff
@@ -0,0 +1,257 @@
+"""2-d Riemann problem (xx minutes)
```

Comment:
Fill in xx.

Reply:
The time estimate is for the default parameters with no additional command line args by the way.